ci-operator/step-registry/openshift/manifests/external/openshift-manifests-external-commands.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+LOCALPATH="${SHARED_DIR}/manifest_external.yaml"
+echo "${SHA256_HASH} -" > /tmp/sum.txt
+curl -fLs "${URL}" | tee "${LOCALPATH}" | sha256sum -c /tmp/sum.txt
```

Comment:
Use long form `--check` for the sha256sum call so it's more clear that it's a check and not a simple calculation

Optionally also do the same for curl with `--fail` `--location` and `--silent`

Reply:
Maybe an error message to show the user the hash that was calculated vs what is expected, something like
```bash
if ! curl -fLs "${URL}" | tee "${LOCALPATH}" | sha256sum -c /tmp/sum.txt >/dev/null 2>/dev/null; then
  echo Expected file at ${URL} to have checksum ${SHA256_HASH} but instead got $(< "${LOCALPATH}" sha256sum | cut -d' ' -f1)
fi

```

Reply:
That looks a bit overcomplicated and doesn't really explain why the hashes are wrong anyway

Reply:
The purpose was to make it easier for the user to fix a wrong hash they calculated manually with the one the CI expects.

And they can verify the file is correct because it's `tee`d

Reply:
Okay, I guess that would be useful